390 Add an API to return session chairs and runners

dpoirier opened the pull request:
Requires an API key; see the API docs once they've
been updated.

dchukhin commented on pycon/schedule/views.py:
```diff
@@ -1,4 +1,4 @@
-import hashlib
+import json
```
is this import used in this file?

dchukhin commented on pycon/schedule/views.py:
```diff
@@ -7,6 +7,9 @@
 from .models import Session, SessionRole
+from pycon.pycon_api.decorators import api_view
+from symposion.schedule.models import Slot
+from symposion.schedule.views import json_serializer
```
is this import used?

dpoirier commented:
Thanks @dchukhin I've removed those unused imports, and I ran flake8 on the files that were modified in this pull and cleaned up the other things it spotted.

dchukhin commented:
looks good :ok:
